src/features/pages/partners/deriv-prime/connectivity-reliability/data.tsx

```diff
@@ -0,0 +1,28 @@
+import { SmartConnectivityTypeItemType } from './types'
+import FixApiIcon from 'images/svg/deriv-prime/fix_api.svg'
```

Comment:
Hi @aswathy-deriv as @prince-deriv suggested we have to import images in other file and we can call them here.

Reply:
This will be addressed once your PR is merged to the feature branch and the image refactoring will be handled there